components/TripForm/index.js

```diff
@@ -11,8 +11,10 @@ import {
 
 import ConfirmationMessage from "../ConfirmationMessage";
 import { useRouter } from "next/router";
+import { useState } from "react";
 
 export default function TripForm() {
+  const [Letters, setLetters] = useState(150);
```

Comment:
Please, write variable names always in camelCase. Here it should be `letters` instead of `Letters`.